src/OptionT.ts

```diff
@@ -113,6 +109,20 @@ export default abstract class OptionT<T> {
    *
    * @param {string} message
    * @returns {T}
+   * @throws a `nullshield:unchecked_unwrap` error if you attempt to call it before first checking
+   * if the [[OptionT]] is a safe to unwrap (in other words, it must be a `Some` value).
+   * @throws a `nullshield:unwrap_on_none` error if you attempt to call it on a `None` value after
+   * first checking that the [[OptionT]] is safe to unwrap.
+   *
+   * ## `nullshield:unchecked_unwrap:` ##
+   * To avoid this issue, either check that the given [[OptionT]] is a `Some` value (with
+   * [[OptionT.isSome]] or [[OptionT.isNone]]) or use [[OptionT.unwrapOr]] instead which allows you
+   * to specify a default value in the case where the given [[OptionT]] is a `None`.
```

Comment:
How about 
> ... or instead use a different function like [[OptionT.unwrapOr]] which allows you to specify a default value, handling the case where the given [[OptionT]] is a `None`.

I like pushing them to `unwrapOr` - its the quick and easy solution to their current problem - but I kinda want to hint that there are other functions as well that could be used to be safer and would maybe express their problems and intent better.

Reply:
I agree.  And since this is in the documentation we've got more room for explanation.

how is something like this:
```
The most direct way to avoid this issue is to either check that the given [[OptionT]] is a `Some` value (with [[OptionT.isSome]] or [[OptionT.isNone]]) or use [[OptionT.unwrapOr]] instead which allows you to specify a default value in the case where the given [[OptionT]] is a `None`.  

However, oftentimes you may not want to simply get the value out of the [[OptionT]]; instead you may want to conditionally use that value in some sort of computation.  In those cases it's likely more clean/clear to use [[OptionT.map]] or a similar function instead.
```

Reply:
@ryanguill no rush, just wanted to make sure you got a notification about this.

Reply:
LGTM